contributing/documentation/overview.rst

```diff
@@ -35,48 +59,79 @@ Let's imagine that you want to improve the installation chapter of the Symfony
+this value accordingly):
 
 .. code-block:: bash
 
     $ cd projects/
     $ git clone git://github.com/<YOUR GITHUB USERNAME>/symfony-docs.git
 
-**Step 3.** Switch to the **oldest maintained branch** before making any change.
-Nowadays this is the ``2.3`` branch:
+**Step 3.** Add the original Symfony docs repository as a "Git remote" executing
+this command:
 
 .. code-block:: bash
 
     $ cd symfony-docs/
-    $ git checkout 2.3
+    $ git remote add upstream https://github.com/symfony/symfony-docs.git
+
+If things went right, you'll see the following when listing the "remotes" of
+your project:
 
-If you are instead documenting a new feature, switch to the first Symfony
-version which included it: ``2.5``, ``2.6``, etc.
+.. code-block:: bash
+
+    $ git remote -v
+    origin  git@github.com:<YOUR GITHUB USERNAME>/symfony-docs.git (fetch)
+    origin  git@github.com:<YOUR GITHUB USERNAME>/symfony-docs.git (push)
+    upstream  https://github.com/symfony/symfony-docs.git (fetch)
+    upstream  https://github.com/symfony/symfony-docs.git (push)
 
-**Step 4.** Create a dedicated **new branch** for your changes. This greatly
-simplifies the work of reviewing and merging your changes. Use a short and
-memorable name for the new branch:
+The purpose of this step is to allow you work simultaneously on the official
+Symfony repository and on your own fork. You'll see this in action in a moment.
+
+**Step 4.** Create a dedicated **new branch** for your changes. Use a short and
+memorable name for the new branch (if you are fixing a reported issue, use
+``fix_XXX`` as the branch name, where ``XXX`` is the number of the issue):
 
 .. code-block:: bash
 
-    $ git checkout -b improve_install_chapter
+    $ git checkout -b improve_install_chapter upstream/2.3
+
+In this example, the name of the branch is ``improve_install_chapter`` and the
+``upstream/2.3`` value tells Git to create this branch based on the ``2.3``
+branch of the ``upstream`` remote, which is the original Symfony Docs repository.
+
+Fixes should always be based on the **oldest maintained branch** which contains
+the error. Nowadays this is the ``2.3`` branch. If you are instead documenting a
+new feature, switch to the first Symfony version which included it: ``2.7``,
+``3.1``, etc.
 
 **Step 5.** Now make your changes in the documentation. Add, tweak, reword and
 even remove any content, but make sure that you comply with the
-:doc:`/contributing/documentation/standards`.
+:doc:`/contributing/documentation/standards`. Then, mark your contents as ready
+to be committed:
+
+.. code-block:: bash
+
+    # if the modified content existed before
+    $ git add book/installation.rst
+    $ git commit book/installation.rst
 
 **Step 6.** **Push** the changes to your forked repository:
 
 .. code-block:: bash
 
-    $ git commit book/installation.rst
     $ git push origin improve_install_chapter
 
+The ``origin`` value is the name of the Git remote which corresponds to your
```

Comment:
that corresponds

`which` and `that` are often interchangeable, but that often sounds stronger (except in certain cases, which I consider quite rare <--- except I just used one there)